support-payment-api/src/main/scala/model/acquisition/AcquisitionDataRowBuilder.scala

```diff
@@ -44,7 +46,7 @@ object AcquisitionDataRowBuilder {
       paymentProvider = Some(paymentProvider),
       printOptions = None,
       browserId = acquisitionData.browserId,
-      identityId = acquisition.identityId.map(_.toString),
+      identityId = acquisition.identityId,
```

Comment:
this was a warning, as identityId already contains a String (I vaguely remember that it was incorrectly modelled as a Long in some places)